ansible/roles/ndh-app/tasks/install_tibco.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: make install files executable
+  ansible.builtin.shell: chmod -R 777 ./*/*.bin
```

Comment:
are you sure you want 777 as the permissions? That means any user could change it and run it.
it'd be best to use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html
and listing those files, OR using 
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html
to find the files, register that list of files, and then apply the file builtin to change the permissions